src/view/com/util/post-embeds/VideoEmbedInner/VideoEmbedInnerNative.tsx

```diff
@@ -69,7 +46,7 @@ export function VideoEmbedInnerNative({
         player={player}
         style={[a.flex_1, a.rounded_sm]}
         contentFit="contain"
-        nativeControls={true}
+        nativeControls={false}
```

Comment:
Also setting this to `false`, which is what it should be. Doesn't affect the visibility of native controls when in fullscreen.